From khali@linux-fr.org Sat Jul  1 08:13:22 2006
Date: Sat, 1 Jul 2006 17:13:37 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: [PATCH 06/13] i2c-ite: Plan for removal
Message-Id: <20060701171337.bbd7d2bf.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-plan-ite-bus-driver-for-removal.patch

Plan the i2c-ite and i2c-algo-ite drivers for removal.
These drivers never compiled since they were added to the kernel
tree 5 years ago. Also see:
http://marc.theaimsgroup.com/?l=linux-mips&m=115040510817448

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/feature-removal-schedule.txt |   11 +++++++++++
 1 file changed, 11 insertions(+)

--- gregkh-2.6.orig/Documentation/feature-removal-schedule.txt
+++ gregkh-2.6/Documentation/feature-removal-schedule.txt
@@ -247,3 +247,14 @@ Why:	The interrupt related SA_* flags ar
 Who:	Thomas Gleixner <tglx@linutronix.de>
 
 ---------------------------
+
+What:	i2c-ite and i2c-algo-ite drivers
+When:	September 2006
+Why:	These drivers never compiled since they were added to the kernel
+	tree 5 years ago. This feature removal can be reevaluated if
+	someone shows interest in the drivers, fixes them and takes over
+	maintenance.
+	http://marc.theaimsgroup.com/?l=linux-mips&m=115040510817448
+Who:	Jean Delvare <khali@linux-fr.org>
+
+---------------------------
